components/org.wso2.carbon.identity.api.server.notification.template/org.wso2.carbon.identity.api.server.notification.template.common/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Invalid license header. Should be WSO2 LLC.

Reply:
check other places as well

Reply:
fixed with https://github.com/wso2/identity-api-server/pull/629/commits/518dbd3e6d4ba60dda9229907bbd9a48f7169ad1